src/Orleans.Core/Properties/AssemblyInfo.cs

```diff
@@ -2,8 +2,10 @@
 using Orleans.CodeGeneration;
 using System.Runtime.CompilerServices;
 
+[assembly: InternalsVisibleTo("Orleans.Azure.Core")]
```

Comment:
this no longer exists, right?
How about `OrleansAzureUtils` below, is it gone too?

Reply:
```OrleansAzureUtils``` still needed for some types, but should be gone when that assembly is gone too.